python_modules/dagster/dagster_tests/components_tests/executable_component_tests/test_pipes_subprocess.py

```diff
@@ -0,0 +1,160 @@
+import inspect
+from textwrap import dedent
+
+from dagster._core.definitions.asset_key import AssetCheckKey, AssetKey
+from dagster._core.definitions.asset_selection import AssetSelection
+from dagster._core.definitions.materialize import materialize
+from dagster._core.definitions.metadata.metadata_value import TextMetadataValue
+from dagster.components.lib.executable_component.subprocess_component import (
+    SubprocessComponent,
+    SubprocessSpec,
+)
+from dagster.components.testing import scaffold_defs_sandbox
+
+
+def test_pipes_subprocess_script_hello_world() -> None:
+    with scaffold_defs_sandbox(component_cls=SubprocessComponent) as sandbox:
+        execute_path = sandbox.defs_folder_path / "script.py"
+        execute_path.write_text("print('hello world')")
+
+        with sandbox.load(
+            component_body={
+                "type": "dagster.components.lib.executable_component.subprocess_component.SubprocessComponent",
+                "attributes": {
+                    "execution": {
+                        "type": "subprocess",
```

Comment:
we can remove this `type: subprocess`  right? repeating it here and in the component type seems less than ideal. Ideally it wouldn't even show up in the schema if it is a fixed value

Reply:
done